src/components/LinkExternal/index.tsx

```diff
@@ -89,7 +92,19 @@ export function LinkExternalBlockExplorer({ chainId, children, className, ...pro
       className={cn("text-foreground hover:no-underline hover:brightness-90", className)}
       {...props}
     >
-      {children ?? displayName}
+      {children ?? (
+        <>
+          {iconUrl && (
+            <img
```

Comment:
Use nextjs image component: 

```suggestion
            <Image
```